ouroboros-network-api/src/Ouroboros/Network/PeerSelection/PeerAdvertise/Type.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE DeriveGeneric #-}
```

Comment:
Let's rename this module to just `Ouroboros.Network.PeerSelection.PeerAdvertise`.